specification/paths/runtime/stick_table_entries.yaml

```diff
@@ -1,4 +1,38 @@
 stick_table_entries:
+  put:
+    summary: Set Entry to Stick Table
+    description: Create or update a stick-table entry in the table.
+    operationId: setStickTableEntries
+    tags:
+      - StickTable
+    parameters:
+      - description: Stick table entry
+        name: stick_table_entry
+        in: body
+        schema:
+          type: object
```

Comment:
I would use here `#/definitions/stick_table_entry` so we have the same objects on PUT and on GET endpoints. Then in the actual `set table` invocation we can execute it for every field which is not `nil`

Reply:
But that means you are providing an API that would set one value for multiple data_types that I think would be useless and also needed validation that at least one of them shouldn't be nil!
It looks strange to me and also because I see some commons in another API for this enum that looks better to me. What's your thought?

Reply:
@mjuraga Also I'm considering the response time as well! Looping over keys is wasting the time while you are putting one value for one key! (As you know sticky table data are usually used for some in-memory procedures and response time in updating it will be really important even ns!)

Reply:
@clwluvw so, why this is designed like this is because one table can have multiple data types for one entry, there are some documentation examples how to define a stick table [here](https://cbonte.github.io/haproxy-dconv/2.6/configuration.html#4.2-stick-table%20type), where you can define a stick-table like this:

```
stick-table type ip size 1m expire 5m store gpc0,conn_rate(30s)
```

Also while fetching table data from the Runtime API you may get multiple values (multiple data types) for one entry with [show table](https://cbonte.github.io/haproxy-dconv/2.6/management.html#show%20table):

```
    $ echo "show table http_proxy" | socat stdio /tmp/sock1
>>> # table: http_proxy, type: ip, size:204800, used:2
>>> 0x80e6a4c: key=127.0.0.1 use=0 exp=3594729 gpc0=0 conn_rate(30000)=1  \
      bytes_out_rate(60000)=187
>>> 0x80e6a80: key=127.0.0.2 use=0 exp=3594740 gpc0=1 conn_rate(30000)=10 \
      bytes_out_rate(60000)=191
```

And the same with [set table](https://cbonte.github.io/haproxy-dconv/2.6/management.html#set%20table):

```
It is possible to pass multiple data_types in a single call
```

Reply:
@mjuraga PTAL

Reply:
ping @mjuraga